german/abwarten/abwarten.md

```diff
@@ -3,4 +3,5 @@ In der Tat lösen sich so einige Probleme von selbst, wenn man einfach nur lange
 Bei genauerer Betrachtung ist die aktuelle allerdings keine solche Situation, in der die sich 
 ergebende Lösung besonders angenehm oder reizvoll erscheint. 
 
-[Daher bietet es sich an doch noch eine andere Entscheidung zu treffen...](../ruderboot.md)
\ No newline at end of file
+[Daher bietet es sich an doch noch eine andere Entscheidung zu treffen...](../ruderboot.md)
+[Oder aber sie warten trotzdem weiter ab...](../weiterwarten.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.